torchtext/datasets/text_classification.py

```diff
@@ -160,10 +178,21 @@ def AG_NEWS(*args, **kwargs):
             Default: 1
         vocab: Vocabulary used for dataset. If None, it will generate a new
             vocabulary based on the train data set.
-        include_unk: include unknown token in the data (Default: False)
+        removed_tokens: removed tokens from output dataset (Default: [])
```

Comment:
Are these changes BC breaking since they'll live outside the experimental folder after all?

Reply:
Could you break out these BC breaking changes, which are unrelated to IMDB, into a separate PR?